### ISSUE TITLE
Small error in the tutorials: Shape 'curve' points

### ISSUE THREAD

**Issue by sdras** [legomushroom/mojs#71]
Working in the tutorials, found what I think is an error- the Shape example for curve is using an object for points:

`var curve = new mojs.Shape({
  shape:        'curve',
  points:       11,
  radius:       25,
  radiusY:      50,
  left:         '50%',
  fill:         'none',
  stroke:       'deeppink',
  isShowStart:   true,
});`

But I don't think the points do anything for the curve (that I've seen). If they do, it would be good to clarify in the text below because it's a little confusing.

Thank you!

**Comment by legomushroom** [comment]
Hi Sarah!

Thanks for the heads up! The `points` field is useless for `curve` shape indeed.
It works only for `zigzag`, `polygon` and `equal` shapes. Is it the `shape tutorial`, right?

**Comment by sdras** [comment]
Yeah, its in the shape tutorial, in the demo with the zigzag and cross :) thanks!

**Comment by sdras** [comment]
Oh! Same applies for cross as well

**Comment by legomushroom** [comment]
thanks!
